core/src/main/scala/sbt/librarymanagement/ScalaArtifacts.scala

```diff
@@ -13,17 +13,22 @@ object ScalaArtifacts {
   final val Scala3CompilerID = "scala3-compiler"
   final val Scala3InterfacesID = "scala3-interfaces"
   final val TastyCoreID = "tasty-core"
+  final val Scala3DocID = "scala3doc"
```

Comment:
Listing this out makes it clear that scala3doc stands out as as oddly named duck.
@romanowski Do you think we can name it `scala3-doc`? - https://github.com/lampepfl/scala3doc/issues/264

Reply:
Maybe this may make sense. We should probably decide between `scaladoc` and `scala3-doc` then. cc @abgruszecki